templates/base.html.twig

```diff
@@ -10,11 +10,17 @@
 	</title>
 	{% block stylesheets %}
 		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
+		<link rel="stylesheet" href="//tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
```

Comment:
This line duplicates the one before it. Did you mean to link to a different stylesheet?